src/app/core/layouts/nav/nav.component.html

```diff
@@ -0,0 +1,13 @@
+<nav class="navbar">
+  <div class="navbar-left">
+    <a href="#">{{ siteName }}</a>
+  </div>
+  <div class="navbar-center">
+    <button pButton type="button" label="{{ 'Start a search' | translate }}" class="text-button" [text]="true" severity="secondary"></button>
```

Comment:
I think you did not understand the value you had to put that will be translated.

Here, instead of 'Start a search', you have to pass the key in the `en.json`, which should be something like that : 

`
"navbar": {
  "startSearch": "Start a search"
}
`


And use in your label : 

`
label="{{ 'navbar.startSearch' | translate }}"
`